source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/Util.java

```diff
@@ -61,21 +61,6 @@ public static void MoveChildrenViews(ViewGroup origin, ViewGroup destination)
         }
     }
 
-    public static BaseInputElement castToInputElement(BaseCardElement cardElement)
```

Comment:
>BaseInputElement [](start = 18, length = 16)

This file probably doesn't belong in this change

Reply:
thanks, I targeted main when I should have targeted release/20.07